krod/core/engine.py

```
@@ -63,48 +70,84 @@ class KrodEngine:
-        # Initialize reasoning systems
-        self.reasoning_system = ReasoningSystem(self.llm_manager, self.config.get("reasoning", {}))
-        self.clarification_system = ClarificationSystem(self.llm_manager, self.config.get("clarification", {}))
-        self.common_sense_system = CommonSenseSystem(self.config.get("common_sense", {}))
+        # Initialize common sense system
+        self.common_sense_system = CommonSenseSystem()
+        
+        # Initialize clarification system
+        self.clarification_system = ClarificationSystem()
+        
+        # Initialize decision system
+        self.decision_system = DecisionSystem()
```

Pylint (3.3.7)

[error] No value for argument 'llm_manager' in constructor call

(E1120)